fig/backends/chronicle/README.md

```diff
@@ -31,7 +34,8 @@ backends=CHRONICLE
    docker run -it --rm \
        -e FALCON_CLIENT_ID="$FALCON_CLIENT_ID" \
        -e FALCON_CLIENT_SECRET="$FALCON_CLIENT_SECRET" \
-       -e GOOGLE_SECURITY_KEY="$GOOGLE_SECURITY_KEY" \
+       -e GOOGLE_SERVICE_ACCOUNT_FILE="$GOOGLE_SERVICE_ACCOUNT_FILE" \
```

Comment:
We may need to pass the service account file as volume (`-v`) option to the docker run command.

Reply:
Makes sense to me, I was building into my local image.